tools/dds/poc/h/poc-h.cpp

```diff
@@ -106,36 +110,61 @@ int main( int argc, char** argv ) try
         }
     }
 
+    LOG_DEBUG( "domain id: " << domain);
+
+    bool create_op_pub_sub = false;
+    if (op_pub_sub_arg.isSet())
+    {
+        create_op_pub_sub = true;
+    }
+
+    bool time_sync_enabled = false;
+    int time_sync_num_of_iterations = 0;
```

Comment:
The above says the default is 10... this is confusing.

Reply:
OK will remove this variable